public/js/components/subscription.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component, PropTypes } from 'react';
+
+import { gettext } from 'utils';
+import * as products from 'products';
+
+
+export default class Subscription extends Component {
+
+  static propTypes = {
+    seller_product: PropTypes.shape({
+      public_id: PropTypes.string,
+    }).isRequired,
+  }
+
+  render() {
+    var productData = products.get(this.props.seller_product.public_id);
+
+    return (
+      <div className="subscription">
+        <div className="product">
+          <img src={productData.img}/>
```

Comment:
Needs an alt attribute. If it doesn't add any value (e.g. it's a dupe of the heading) make it an empty string.